great_expectations/util.py

```diff
@@ -1804,6 +1807,7 @@ def get_context(  # noqa: PLR0913
     cloud_access_token: str | None = None,
     cloud_organization_id: str | None = None,
     cloud_mode: bool | None = None,
+    mode: str | None = None,
```

Comment:
```suggestion
    mode: Literal["ephemeral", "file", "cloud"] | None = None,
```
This ensures better auto-complete and IDE help

Reply:
ty!